runtime/gc_modron_startup/mmparseXXgc.cpp

```diff
@@ -933,6 +933,40 @@ gcParseXXgcArguments(J9JavaVM *vm, char *optArg)
 			continue ;
 		}
 
+		if (try_scan(&scan_start, "adaptiveThreadingSensitivityFactor=")) {
+			UDATA sensitivityFactor = 0;
+			if (!scan_udata_helper(vm, &scan_start, &sensitivityFactor, "adaptiveThreadingSensitivityFactor=")) {
+				returnValue = JNI_EINVAL;
+				break;
+			}
+			extensions->adaptiveThreadingSensitivityFactor = ((float)sensitivityFactor) / 10.0f;
+
+			continue ;
+		}
+
+		if (try_scan(&scan_start, "adaptiveThreadingWeightActiveThreads=")) {
+			UDATA adaptiveThreadingWeightActiveThreads = 0;
+			if (!scan_udata_helper(vm, &scan_start, &adaptiveThreadingWeightActiveThreads, "adaptiveThreadingWeightActiveThreads=")) {
+				returnValue = JNI_EINVAL;
+				break;
+			}
+
+			extensions->adaptiveThreadingWeightActiveThreads = ((float)adaptiveThreadingWeightActiveThreads) / 100.0f;
+
+			continue ;
+		}
+
+		if (try_scan(&scan_start, "adaptiveThreadBooster=")) {
+			UDATA adaptiveThreadBooster = 0;
+			if (!scan_udata_helper(vm, &scan_start, &adaptiveThreadBooster, "adaptiveThreadBooster=")) {
+				returnValue = JNI_EINVAL;
+				break;
+			}
+			extensions->adaptiveThreadBooster = (float)adaptiveThreadBooster;
```

Comment:
no divide by 100?

Reply:
Nope, this is not a percentage, it's an amount to add on top of calculated value

Reply:
Yeah, but you are limited to only integers. Default is 0.85. How do you specify 1.1 or 0.7? My though was you specify 110 or 70  and then divide by 100.

Reply:
You're right, it should be changed then

Reply:
Btw, there are ways to provide floats as well (for example -Xmaxf0.8), but this is equally good.